opm/parser/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -43,16 +43,17 @@
 #include <opm/parser/eclipse/EclipseState/SimulationConfig/SimulationConfig.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/TableManager.hpp>
 #include <opm/parser/eclipse/Parser/ParserKeywords/M.hpp>
+#include <opm/parser/eclipse/Parser/ParseContext.hpp>
 #include <opm/parser/eclipse/Units/Dimension.hpp>
 #include <opm/parser/eclipse/Units/UnitSystem.hpp>
 #include <opm/parser/eclipse/Parser/MessageContainer.hpp>
 
 
 namespace Opm {
 
-    EclipseState::EclipseState(std::shared_ptr<const Deck> deck, const ParseContext& parseContext) :
+    EclipseState::EclipseState(std::shared_ptr<const Deck> deck, const ParseContext parseContext) :
```

Comment:
It's pretty pointless to pass something as a const copy (because the copy is local anyway, and modifications won't bubble outwards).